cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -181,6 +181,54 @@ class ReadFileTest {
         assertEquals(ion.singleValue(expected), actual)
     }
 
+    @Test
+    fun readPostgreSQLTextFile() {
+        writeFile("simple_postgresql.txt", "id\tname\tbalance\n1\tBob\t10000.00")
+
+        val args = listOf("\"${dirPath("simple_postgresql.txt")}\"", "{type:\"postgresql_text\", header:true}").map { it.exprValue() }
+
+        val actual = function.call(env, args).ionValue
+        val expected = "[{id:\"1\",name:\"Bob\",balance:\"10000.00\"}]"
+
+        assertEquals(ion.singleValue(expected), actual)
+    }
+
+    @Test
+    fun readMySQLCsvFile() {
+        writeFile("simple_mysql.csv", "id\tname\tbalance\n1\tBob\t10000.00")
+
+        val args = listOf("\"${dirPath("simple_mysql.csv")}\"", "{type:\"mysql_csv\", header:true}").map { it.exprValue() }
+
+        val actual = function.call(env, args).ionValue
+        val expected = "[{id:\"1\",name:\"Bob\",balance:\"10000.00\"}]"
+
+        assertEquals(ion.singleValue(expected), actual)
+    }
+
+    @Test
+    fun readMongodbCsvFile() {
+        writeFile("simple_mongo.csv", "id,name,balance\n1,Bob,10000.00")
+
+        val args = listOf("\"${dirPath("simple_mongo.csv")}\"", "{type:\"mongodb_csv\", header:true}").map { it.exprValue() }
+
+        val actual = function.call(env, args).ionValue
+        val expected = "[{id:\"1\",name:\"Bob\",balance:\"10000.00\"}]"
+
+        assertEquals(ion.singleValue(expected), actual)
+    }
+
+    @Test
+    fun readMongodbTsvFile() {
+        writeFile("simple_mongo.tsv", "id\tname\tbalance\n1\tBob\t10000.00")
+
+        val args = listOf("\"${dirPath("simple_mongo.tsv")}\"", "{type:\"mongodb_tsv\", header:true}").map { it.exprValue() }
+
+        val actual = function.call(env, args).ionValue
+        val expected = "[{id:\"1\",name:\"Bob\",balance:\"10000.00\"}]"
+
+        assertEquals(ion.singleValue(expected), actual)
+    }
```

Comment:
I was browsing `CSVFormat`'s [documentation](https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.html#MONGODB_CSV) to see more of the differences between the different CSV parsing formats. I noticed that for `mongodb_csv` and `mongodb_tsv` that it states:

> **Parsing is not supported yet.**

Could we verify if the parsing is actually supported by passing a CSV/TSV created by `mongoexport`? If it's actually not supported, I think we should remove those options from the CLI parsing.

Reply:
I was using MongDB Commpass to create the CSV file. It just looks like a normal CSV file. 
[test.csv](https://github.com/partiql/partiql-lang-kotlin/files/7825251/test.csv)

If the `CSVForamt` docs says it is not supported yet. I think we should remove them.